lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -97,28 +96,30 @@ class DOMSize extends Audit {
       {key: 'value', itemType: 'numeric', text: str_(UIStrings.columnValue)},
     ];
 
+    const i18n = new I18n(context.settings.locale);
```

Comment:
it's a bummer these won't get into the localeLog, maybe we file a separate issue to track that?

Reply:
no idea what this means :)

Reply:
Oh sorry I meant [icuMessagePaths](https://github.com/GoogleChrome/lighthouse/blob/39b13bc4f48cf6933f28f3ade87e1705f18a2ddb/lighthouse-core/runner.js#L152) 

when we create i18n'd strings using `str` we get to track it in the lhr so we can use `swap-locale` later, it'd be great if these usages could take advantage of that

Reply:
ah right. an additional challenge - besides figuring out how to sink these paths to a central place - the swap locale script only works for UIStrings, not for value formatting.

Instead, I think we shouldn't format in the audit at all. These values should be raw numbers - and all the `numeric` detail types should be formatted by the report renderer. That way there's nothing to swap.

Reply:
does that make sense?

Reply:
oh sorry! yes! 

> I think we shouldn't format in the audit at all. These values should be raw numbers - and all the numeric detail types should be formatted by the report renderer (odd that this isn't the case now ...). That way there's nothing to swap.

Agreed I think there was even a todo I put in one of my PRs during review on this topic that I've totally forgotten to merge...

Reply:
Cool.. so I'll leave the changes in the audits as they are now. It's still an  improvement. We can follow up on formatting in just the renderer later.

I'll fix those tests now..